README.md

```diff
@@ -5,29 +5,43 @@
 EdgePi thermocouple node that reads temperature on input received.
 
 ## Install
+
 Install normally through the node-red editor or install with npm in your node-red directory
-(typically located  at `~/node.red`) by running the following command:
+(typically located at `~/node.red`) by running the following command:
+
 ```
 npm install @edgepi-cloud/node-red-edgepi-digital-thermocouple
 ```
 
 ### Properties
+
 - **RPC Server**: The connection to your EdgePi's RPC Server.
 - **Output**: What temperature reading(s) will be sent out through the flow.
 
 ### Inputs
-Any message can be used to trigger this node.
 
-### Outputs
-- When configured to read only linearized temperature:
-  - **payload** *number*: Regular temperature reading.
-  
-- When configured to read only cold junction temperature:
-  - **payload** *number*: Temperature reading of the cold junction sensor.
-  
-- When configured to read both cold junction and linearized temperature:
-  - **payload** *array[coldJunctionTemp, linearizedTemp]*: An array containing both the cold junction and linearized temperature.
+- **payload** _string_: The output type. 'lin' for Linearized Temperature, 'cj' for Cold-Junction
+  Temperature, 'cj-lin' for Cold Junction & Linearized Temperature.
 
+Example input:
+```
+msg {
+  "payload": "cj-lin"
+}
+```
 
-**NOTE:** Currently, EdgePi nodes are only available on x86 systems.
+### Outputs
 
+- When configured to read linearized temperature:
+  - **payload** _number_: Regular temperature reading.
+- When configured to read cold junction temperature:
+  - **payload** _number_: Temperature reading of the cold junction sensor.
+- When configured to read both cold junction and linearized temperature:
+  - **payload** _array[coldJunctionTemp, linearizedTemp]_: An array containing both the cold junction and linearized temperature.
+
+Example output:
+```
+msg {
+  "payload": [24.1875, 23.0703125]
+}
```

Comment:
let's simplify this. I don't think we need to pass this kind of information on what to read. @sjpark608 does it make sense to have any inputs for TC?

Reply:
I think it doesn't make sense to choose which temperature to read. The end user should be only interested in reading the linearized temperature

Reply:
In that case, should I just ignore this pr? Besides adding input options, this pr doesn't include anything significant.

Reply:
In that case yes we should ignore the PR

Reply:
Reverted